great-idea/contact.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea! - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
```

Comment:
Let's start separating css into separate files for each page. You should think about having a file called index.css (home page), services.css, and contact.css. Even if you're reusing some styles across multiple pages, you should copy them and separate them into new files. It will make your life so much easier!